### DISCUSSION TITLE
Facades registration does not work? as I want to customize the Facade name is it even possible?

### DISCUSSION THREAD

**Discussion by msonowal** [spatie/laravel-package-tools#59] [Q&A]
Facades registration does not work? as I want to customize the Facade name is it even possible?

**Comment by oliverbj** [comment]
I can't get this to work either

**Reply by cxj** [reply]
I've also had no luck getting my package's Facade registered using this Spatie package.  I'm about to completely remove it and do it all by hand, as it seems to be the only way to make it work.  🙁
